src/gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -18249,7 +18251,15 @@ properties:
         - Brain, Parietal
         - Brain, Temporal lobe
         - Brain, Ventricle, NOS
+        - Breast, Left Lower Inner
+        - Breast, Left Lower Outer
+        - Breast, Left Upper Inner
+        - Breast, Left Upper Outer
         - Breast, NOS
+        - Breast, Right Lower Inner
+        - Breast, Right Lower Outer
+        - Breast, Right Upper Inner
+        - Breast, Right Upper Outer
```

Comment:
just noting the capitalization here is different from the ticket (but i think the one here is correct)